app/code/Magento/Store/Model/StoreManager.php

```diff
@@ -114,23 +117,25 @@ public function __construct(
     }
 
     /**
-     * {@inheritdoc}
+     * @inheritdoc
      */
     public function setCurrentStore($store)
     {
         $this->currentStoreId = $store;
     }
 
     /**
-     * {@inheritdoc}
+     * @inheritdoc
      */
     public function setIsSingleStoreModeAllowed($value)
     {
         $this->isSingleStoreAllowed = $value;
     }
 
     /**
-     * {@inheritdoc}
+     * Check if store has only one store view
```

Comment:
Please revert the comment and just remove curly braces `{}`. Please refer to the guide https://devdocs.magento.com/guides/v2.3/coding-standards/docblock-standard-general.html#inheritdoc